core/templates/dev/head/pages/exploration_player/SupplementalCardDirective.js

```diff
@@ -82,6 +82,12 @@ oppia.directive('supplementalCard', [
             return maxHelpCardHeightSeen > $(window).height() - 100;
           };
 
+          $scope.getHelpCardBottomPosition = function() {
+            var helpCard = $('.conversation-skin-help-card');
+            var container = $('.conversation-skin-supplemental-card-container');
+            return Math.max(container.height() - helpCard.height() / 2, 0);
```

Comment:
@nithusha21 I am working on migrating this directive but failing to understand this formula.
Can you please answer the following questions?

1. What is the ideal position of the help card. Where should it be shown to the user?
2. How this formula achieves the expected positioning?

Reply:
I don't remember to be frank. If I did actually make this change, it was probably because the help card was getting hidden below the footer, which made the continue button not clickable. To fix this we moved it further up. Ideal position being bottom right, but above footer (make sure to check different screen sizes)

Reply:
Thanks, I have got my answer now.